OpenRA.Mods.Common/UpdateRules/Rules/20180923/ExtractHackyAIModules.cs

```diff
@@ -14,15 +14,15 @@
 
 namespace OpenRA.Mods.Common.UpdateRules.Rules
 {
-	public class AddHarvesterBotModule : UpdateRule
+	public class ExtractHackyAIModules : UpdateRule
 	{
-		public override string Name { get { return "Split HackyAI harvester handling to HarvesterBotModule"; } }
+		public override string Name { get { return "Split HackyAI logic handling to BotModules"; } }
 		public override string Description
 		{
 			get
 			{
-				return "Some properties and all harvester handling have been moved from HackyAI\n" +
-					"to the new HarvesterBotModule.";
+				return "All properties and most logic are being moved from HackyAI\n" +
```

Comment:
This is now "Most properties and logic" rather than "All properties and most logic" thanks to `MinOrderQuotientPerTick`.